src/DynExpManager/Instruments/NP_Conex_CC.h

```diff
@@ -0,0 +1,396 @@
+		class MoveAbsoluteTask final : public DynExp::TaskBase
+		{
+		public:
+			MoveAbsoluteTask(PositionerStageData::PositionType Position, CallbackType CallbackFunc) noexcept
+				: TaskBase(CallbackFunc), Position(Position) {}
+
+		private:
+			virtual DynExp::TaskResultType RunChild(DynExp::InstrumentInstance& Instance) override;
+
+			const PositionerStageData::PositionType Position;
+		};
+		///@}
+
+		class MoveAbsoluteExecutionTask final : public DynExp::TaskBase
+		{
+		public:
+			MoveAbsoluteExecutionTask(PositionerStageData::PositionType Position, CallbackType CallbackFunc, std::chrono::system_clock::time_point DeferUntil = {}) noexcept
+				: TaskBase(CallbackFunc, DeferUntil), Position(Position) {}
+
+		private:
+			virtual DynExp::TaskResultType RunChild(DynExp::InstrumentInstance& Instance) override;
+
+			const PositionerStageData::PositionType Position;
+
+		};
+
+		///@{
+		/// @brief In this task, some commands cannot be send right after each other to the controller. There has to be some waiting time in between. 
+		/// Therefore, this tasks calls the subtasks StopMotionTask and MoveRelativeExecutionTask in this order with a delay in between.
+		class MoveRelativeTask final : public DynExp::TaskBase
+		{
+		public:
+			MoveRelativeTask(PositionerStageData::PositionType Position, CallbackType CallbackFunc) noexcept
+				: TaskBase(CallbackFunc), Position(Position) {}
+
+		private:
+			virtual DynExp::TaskResultType RunChild(DynExp::InstrumentInstance& Instance) override;
+
+			const PositionerStageData::PositionType Position;
+		};
+		///@}
+
+		class MoveRelativeExecutionTask final : public DynExp::TaskBase
+		{
+		public:
+			MoveRelativeExecutionTask(PositionerStageData::PositionType Position, CallbackType CallbackFunc, std::chrono::system_clock::time_point DeferUntil = {}) noexcept
+				: TaskBase(CallbackFunc, DeferUntil), Position(Position) {}
+
+		private:
+			virtual DynExp::TaskResultType RunChild(DynExp::InstrumentInstance& Instance) override;
+
+			const PositionerStageData::PositionType Position;
+
+		};
+
+		class StopMotionTask final : public DynExp::TaskBase
+		{
+			virtual DynExp::TaskResultType RunChild(DynExp::InstrumentInstance& Instance) override;
+		};
+	}
+
+	class NP_Conex_CC_StageData : public PositionerStageData
+	{
+		friend class NP_Conex_CC_Tasks::InitTask;
+		friend class NP_Conex_CC_Tasks::UpdateTask;
+
+	public:
+		using ChannelType = int16_t;
+
+		/**
+		 * @brief Interprets and queries the controller's internal state based on the TS command byte code.
+		 */
+		struct Conex_CCStatusType
+		{
+			constexpr void Set(uint8_t ByteCode) noexcept { this->ByteCode = ByteCode; }
+
+			/** @name Newport Conex-CC controller states
+			 * These methods identify the current internal state of the CONEX-CC motion controller,
+			 * based on the `ByteCode` value returned by the `TS` command.
+			 */
```

Comment:
Nice, detailed comments. Please follow the doxygen syle used in other files in the future. Note that there should not be a space before the `*/` symbol.